packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -406,14 +402,45 @@ export async function getLinkRemovalByDidExtrinsic(
 /**
  * Return the default sign callback, which uses the address argument to crete a signing closure for the given payload.
  *
- * @param keyring The keyring to retrieve the signing key.
+ * @param keypair The keypair to sign the data with.
  * @returns The signature generating callback that uses the keyring to sign the input payload using the input address.
  */
-export function defaultSignerCallback(keyring: Keyring): LinkingSignerCallback {
-  return (payload: HexString, address: Address): Promise<HexString> =>
-    Promise.resolve(
-      u8aToHex(keyring.getPair(address).sign(payload, { withType: false }))
-    )
+export function makeLinkingSignCallback(
```

Comment:
I changed the name since the function is not a callback by itself, it makes callbacks.